src/pages/admins.js

```diff
@@ -0,0 +1,106 @@
+var shortid = require('shortid')
+
+function create(templates, division, admin_group, req, res) {
+  if (!req.user || !req.user.isAdmin) {
+    res.sendStatus(403)
+    return
+  }
+
+  division.getDivisions().then(divisions => {
+    admin_group.getAdminGroups().then(admin_groups => {
+      var html = templates.admin.edit({
+        user: req.user,
+        verb: 'Create',
+        divisions: divisions,
+        admin_groups: admin_groups
+      })
+
+      res.send(html)
+    })
+  })
+}
+
+function list(templates, admin, req, res) {
+  if (!req.user || !req.user.isAdmin) {
+    res.sendStatus(403)
+    return
+  }
+
+  admin.getAdmins().then(admins => {
+    var html = templates.admin.list({
+      user: req.user,
+      admins: admins
+    })
+
+    res.send(html)
+  }).catch(err => {
+    console.error(err)
+    res.sendStatus(500)
+  })
+}
+
+function edit(templates, admin, division, admin_group, req, res) {
+  if (!req.user || !req.user.isAdmin) {
+    res.sendStatus(403)
+    return
+  }
+
+  var admin_id = req.params.admin_id
+
+  admin.getAdmins({ admin_id: admin_id }).then(([admin]) => {
+    division.getDivisions().then(divisions => {
+      admin_group.getAdminGroups().then((admin_groups) => {
+        var html = templates.admin.edit({
+          user: req.user,
+          verb: 'Edit',
+          admin: admin,
+          divisions: divisions,
+          admin_groups: admin_groups
+        })
+        res.send(html)
+      }).catch(err => {
+        console.error(err)
+        res.sendStatus(500)
+      })
+    })
+  })
+}
+
+function post(admin, req, res) {
```

Comment:
Similar to groups, looks as though admins might be able to edit admins above their group.